content/en/about/day-in-life-dev.md

```diff
@@ -6,95 +6,111 @@ position: 7
 category: 'About Us'
 ---
 
-Potential candidates often wonder what a typical day looks like at Narrative, and this page is an attempt to answer the question.
+Potential candidates often wonder what a typical day looks like at Narrative, and this page is an attempt to answer the
+question.
 
 ## Sync with the rest of the team
+
 ### \#daily-gsd
 
-We use [Slack](https://slack.com/) to stay connected with the rest of the team. 
+We use [Slack](https://slack.com/) to stay connected with the rest of the team.
 
-The `#daily-gsd` channel lets us keep track of what our teammates are working on and identify blockers as well as potential conflicts.
-Each one of us starts the day by taking a look at his/her work queue, what (s)he achieved yesterday, and what (s)he plans to complete today, 
-and writes a 2-3-liner in `#daily-gsd` to let the rest of the team know about the current priorities.
+The `#daily-gsd` channel lets us keep track of what our teammates are working on and identify blockers as well as
+potential conflicts.
+Each of us starts the day by taking a look at our work queue, what we achieved yesterday, and what we plan to complete
+today, then we write a couple sentences in `#daily-gsd` to let the rest of the team know about the current priorities.
 
 ### \#auto-techops
 
-The person who is on techops rotation will look at the various warnings and alerts in the `#auto-techops` channel in order to identify 
-things that need to be fixed, failed clusters, etc.
-
-### \#partner-success and \#helpdesk
+The person who is on call will look at the various warnings and alerts in the `#auto-techops` channel in order to
+identify things that need to be fixed, failed clusters, etc.
 
-The person who is on Partner Success (PS) rotation will monitor the `#partner-success` and `#helpdesk` channels, as well as the Partner 
-Success queue in [Clubhouse](http://clubhouse.io/) in order to determine the next priorities.
+The on call will also monitor the `#partner-success` and `#helpdesk` channels to prevent partners and the business from
+beingn blocked on technical issues.
 
-There is also a 30-minute meeting on Monday with the sales team to proritize the work and get an idea about the upcoming, sales/partner-driven
-requests.
+There is also a 30-minute meeting on Monday with the sales team to proritize the work and get an idea about the
+upcoming sales/partner-driven requests.
 
-### Crucible reviews
+### Code Reviews
```

Comment:
Update: got rid of mentions of Crucible.

This section feels a bit out of date, I don't think we have to motivate code reviews anymore and no one does code review meetings. But left it as is.